### PR TITLE
Fix a test for lfortran executable on Windows

### LINKED ISSUE #17: The `lfortran.exe` executable fails to run on Windows
The error one gets is:

Screenshot from 2020-10-07 13-58-19

```
lfortran.exe - Entry Point Not Found
The procedure entry point could not be located in the dynamic link library c:\...\lfortran.exe
```

### PR DESCRIPTION
**Pull request by certik**
And pin xeus, to ensure ABI compatibility (this was the main bug).

Fixes #17.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by certik** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,10 +29,10 @@ requirements:
 
 test:
   commands:
-    - test -f ${PREFIX}/bin/lfortran  # [unix]
-    - if exist %LIBRARY_PREFIX%\bin\lfortran.exe (exit 0) else (exit 1)  # [win]
     - lfortran --help
     - lfortran --version
+    - test -f ${PREFIX}/bin/lfortran  # [unix]
+    - if exist %LIBRARY_PREFIX%\bin\lfortran.exe (exit 0) else (exit 1)  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml:35]:
```suggestion
    - if not exist %LIBRARY_PREFIX%\bin\lfortran.exe exit 1  # [win]
```

**Comment by certik** [comment]
Thanks @isuruf !

**Comment by certik** [comment]
Thanks Isuru for your help. How did you figure out it was `xeus` causing the ABI incompatibility?

**Comment by isuruf** [comment]
Dependency walker.

image

**Comment by certik** [comment]
Nice! Indeed, it shows some missing symbols or something. 

Do you have a local Windows machine where you debug such things? I used Appveyor remote login to see what is going on at #17.

**Comment by isuruf** [comment]
> Do you have a local Windows machine where you debug such things?

Yes, I have access to a windows machine on the cloud sponsored by OVH

**Pull request merged**